src/refda/exec.h

```diff
@@ -54,10 +56,25 @@ int refda_exec_run_seq(refda_exec_seq_t *seq);
 /** Work thread function for the Agent execution manager.
  *
  * @param[in] arg The context ::refda_agent_t pointer.
- * @return Always NULL pointer.
```

Comment:
This should be left in; it is part of the pthread API needs.

Reply:
Agree, added back.